expression/chunk_executor.go

```diff
@@ -125,11 +125,12 @@ func evalOneVec(ctx sessionctx.Context, expr Expression, input *chunk.Chunk, out
 			i64s := result.Int64s()
 			buf := chunk.NewColumn(ft, input.NumRows())
 			buf.ReserveBytes(input.NumRows())
+			byteSize := (ft.Flen + 7) >> 3
```

Comment:
Could anyone explain to me why we use `byteSize := (ft.Flen + 7) >> 3` here?

Reply:
And I find two other places using `(ft.Flen + 7) >> 3` also. Should we add a comment for it? Maybe there are people who don’t understand it, just like me.

Reply:
In bit Type, Flen represents the number of bits, byteSize := (ft.Flen + 7) >> 3 is to convert the number of bits to the number of bytes.@mmyj

Reply:
Thank you for your explain, and what is the effect of `(+7)>>3`?

Reply:
>> 3 is the same as dividing by 8,
+7 is to prevent the number of bits that cannot be divisible by 8 causing the resulting bytes space is not enough to store the original data
for example if ft.Flen = 23，23/8 = 2, it's 16 bytes is not enough to store 20 bytes,but (23+7)/8 = 3,it's ok @mmyj

Reply:
Thanks